### ISSUE TITLE
vvp: Handle more host signals when -n is absent

### ISSUE THREAD

**Issue by mcandre** [steveicarus/iverilog#543]
Would be helpful for the vvp automatically finish when Control+C (SIGINT) or Control+D (EOF) are supplied to the REPL. Similar to how Control+C (SIGINT) works with the -n flag.

Then the vvp interpreter would behave like other common interpreters.

**Comment by caryr** [comment]
$finish is the normal way to exit from the command prompt/REPL. CTRL+C would be slightly problematic since that is how you get to the command prompt and typing it twice could cause you to inadvertently exit the simulation. CTRL+D is a possibility, but how is that any better than typing a few more characters for $finish?

**Comment by mcandre** [comment]
Agreed, Ctrl+D / EOF is reasonable but maybe not Ctrl+C / SIGINT here.
